pkg/model/master_volumes.go

```diff
@@ -160,9 +163,16 @@ func (b *MasterVolumeBuilder) addAWSVolume(c *fi.ModelBuilderContext, name strin
 	}
 	if volumeType == "io1" {
 		t.VolumeIops = i64(int64(volumeIops))
+
+		// https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html
```

Comment:
So it would probably be better to do this validation earlier - you will only get told about this when you do `kops update cluster`, but it would be nice to tell people earlier.  (Just as k8s, validates objects and tries to prevent invalid changes - for example you can't update pod limits to a negative resource value ... I think!).

In kops, this would mean putting it here: https://github.com/kubernetes/kops/blob/master/pkg/apis/kops/validation/aws.go#L39

But it's fine to have both validations, and it's a bit more tricky to add it to the validation because the etcd validation is currently spread out across a few files and not comprehensive (as you've found!)  So we can leave this here.

I do have another concern, which is that we're hard-coding an AWS limit, and that limit might change, but we can add it and see if/when it changes.  It likely won't change too often :-)